invenio_app_ils/documents/mappings/os-v1/documents/document-v2.0.0.json

```
@@ -84,7 +90,14 @@
             "type": "keyword"
           },
           "value": {
-            "type": "keyword"
+            "type": "text",
+            "analyzer": "custom_analyzer",
+            "search_analyzer": "custom_analyzer",
+            "fields": {
+              "keyword": {
+                "type": "keyword"
+              }
+            }
```

normally, in order to avoid problems with reindexing, this could be done in opposite way

```suggestion
            "type": "keyword",
            "fields": {
              "text": {
                "type": "text",
                "analyzer": "custom_analyzer",
                "search_analyzer": "custom_analyzer",
              }
            }
```

in this way you add a new field instead of changing the type of the existing one